code/localization/localize.cpp

```diff
@@ -62,7 +62,7 @@ bool *Lcl_unexpected_tstring_check = nullptr;
 // NOTE: with map storage of XSTR strings, the indexes no longer need to be contiguous,
 // but internal strings should still increment XSTR_SIZE to avoid collisions.
 // retail XSTR_SIZE = 1570
-// #define XSTR_SIZE	1672
+// #define XSTR_SIZE	1675
```

Comment:
This says you added 3 XSTR's, but I only counted 2 additions in your changes.  1672 seems to have been skippd?

Reply:
1672 is the "or" in controlsconfigcommon.cpp

Reply:
Oh, there it is.  It didn't get the highlighting I was expecting, because the lines were completely new. 🙄